src/components/Download/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { type CommonProps } from 'types/components';
+import EContainer from '../EContainer';
+import { Button, Typography, Divider, Stack } from '@mui/material';
+import DownloadIcon from '@mui/icons-material/Download';
+
+interface DownloadItem {
+  label: string;
+  fileName?: string;
+  href: string;
+}
+
+export interface IDownolad extends CommonProps {
+  title: string;
+  subtitle?: string;
+  type?: 'button' | 'link';
+  items: DownloadItem[];
+}
+
+const Download = (props: IDownolad) => {
+  const { title, subtitle, items, theme, type } = props;
+  const backgroundColor = theme === 'dark' ? 'primary.dark' : 'white';
+  const textColor = theme === 'dark' ? 'primary.contrastText' : 'text.primary';
+  const isTypeButton = type === 'button';
+  return (
+    <EContainer direction="column" background={backgroundColor} py={10}>
+      <Typography variant="h2" mb={1} color={textColor}>
+        {title}
+      </Typography>
+      {subtitle && (
+        <Typography variant="body1" mb={1} color={textColor}>
+          {subtitle}
+        </Typography>
+      )}
+      <Divider />
+      <Stack alignItems={'flex-start'}>
```

Comment:
We've to be foresighted : the `alignItems` could be decided by the designer (start, center, end).
So, please add this possibility otherwise leave `start-end` as default

Reply:
By Design they didn't added this layout configuration, but no problem to provide it